### DISCUSSION TITLE
Proof

### DISCUSSION THREAD

**Discussion by Lolly1150** [community/community#17694] [GitHub Education]
Hi, I’m applying for github student and as proof I would like to put my school report card what kind of proof should I put?

**Comment by mlarasusan** [comment]
Hello [Lolly1150](https://github.com/Lolly1150). For more information about the application process and what documents we accept, please read our [blog post](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/). Thank you :]
